packages/react/src/components/badge/Badge.tsx

```diff
@@ -35,9 +35,8 @@ export class Badge extends React.Component<IBadgeProps> {
         return (
             <span className={this.className} aria-label="badge">
                 {'icon' in this.props ? (
-                    <Svg
-                        svgName={this.props.icon}
-                        svgClass={classNames('icon', {'mod-16': !this.isSmall, 'mod-12': this.isSmall})}
+                    <this.props.icon
+                        height={this.isSmall ? 14 : 18}

```

Comment:
14 and 18 are pixels values, they are what looked better in the context of an icon inside the badge, I doubt it will be useful elsewhere

Reply:
The typescript type for the height prop makes it clearer when you are in your IDE

Reply:
https://github.com/coveo/plasma/pull/2493/files#diff-db1de6304136f4c8bfe9e98c811ce9ef3222aa937324b8371727216b441862afR13

So you use 14, 16 and 18.

It would be better for me to have 3 constants SIZE_SMALL, SIZE_MEDIUM, SIZE_LARGE to give context to this size.

Reply:
What about when I will need `24`, `32`, `48`, `56`, `64`, etc... having constants will just make it harder to know what number of pixels it represents under the hood

Reply:
I will merge the PR for now since this one is blocking me from doing other PRs, but this is merged in `next` branch not `master` so we can come back to it later if needed